src/components/ChangelogDialog/__stories__/ChangelogDialog.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
+
+import {Button} from '../../Button';
+import {ChangelogDialog, ChangelogDialogProps} from '../ChangelogDialog';
+import {ChangelogItem} from '../types';
+
+export default {
+    title: 'Components/ChangelogDialog',
+    component: ChangelogDialog,
+} as Meta;
+
+const items: ChangelogItem[] = [
+    {
+        date: '03 Jul 2022',
+        isNew: true,
+        version: '2.0.0',
+        title: 'New navigation',
+        image: {
+            src: 'https://jing.yandex-team.ru/files/msitnikov1/pic1%402x.png',
```

Comment:
This (and other references to jing) should be replaced with publicly available image

Reply:
Done